aten/src/ATen/native/quantized/cuda/make_quantized_tensor.cu

```diff
@@ -4,17 +4,10 @@
 namespace at {
 namespace native {
 
-Tensor make_per_tensor_quantized_tensor_cuda(
-    const Tensor& self,
-    double scale,
-    int64_t zero_point) {
-  Tensor dst = at::_empty_affine_quantized(
-      self.sizes(),
-      self.options().dtype(toQIntType(self.scalar_type())),
-      scale,
-      zero_point);
+static inline void assign_quantized_tensor_cuda(
```

Comment:
This should fix
`On Windows, the enclosing parent function for an extended __host__ __device__ lambda cannot have internal or no linkage`
```suggestion
inline void assign_quantized_tensor_cuda(
```